_locales/en/messages.json

```diff
@@ -646,6 +646,12 @@
     "configurationBatteryScale": {
         "message": "Voltage Scale"
     },
+    "configurationVbatPidCompensation": {
+        "message": "Vbat PID Compensation"
+    },
+    "configurationVbatPidCompensationHelp": {
+        "message": "Increase the PID values to compensate when Vbat gets lower. This will give more constant flight characteristics throughout the flight."
```

Comment:
https://github.com/betaflight/betaflight/blob/master/src/main/flight/mixer.c#L763
Op 13 jul. 2016 01:25 schreef "Michael Keller" <notifications@github.com>:

> In _locales/en/messages.json
> <https://github.com/betaflight/betaflight-configurator/pull/111#discussion_r70540910>
> :
>
> > @@ -646,6 +646,12 @@
> >      "configurationBatteryScale": {
> >          "message": "Voltage Scale"
> >      },
> > +    "configurationVbatPidCompensation": {
> > +        "message": "Vbat PID Compensation"
> > +    },
> > +    "configurationVbatPidCompensationHelp": {
> > +        "message": "Increase the PID values to compensate when Vbat gets lower. This will give more constant flight characteristics throughout the flight."
>
> I can't find a reference to calculateBatteryPercentage. I must really be
> seeing things. :-(
>
> ~/git/betaflight$ grep calculateBatteryPercentage src/main/flight/mixer.c
> ~/git/betaflight$
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/betaflight/betaflight-configurator/pull/111/files/c63af70999b71270e4c2c6b4d60295356c402a65#r70540910>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKQlhM4RXFy_2f70G4NHtrkQ40jvOpRYks5qVCJrgaJpZM4JKUzM>
> .
>